edge-auth/main.go

```diff
@@ -30,6 +30,7 @@ func main() {
 	var oauthClientSecretPath string
 
 	var writeDebug bool
+	secureCookie := false
```

Comment:
Default to true, unless set to false.

Reply:
This was left default false so a user upgrading their OFC installation by just bumping the img tag would get the same functionality. 

This is because the env var may not be set/present?